tests/operator_test.go

```diff
@@ -438,6 +438,23 @@ var _ = Describe("ALL Operator tests", func() {
 				err = f.CdiClient.CdiV1beta1().CDIs().Delete(context.TODO(), cr.Name, metav1.DeleteOptions{DryRun: []string{"All"}})
 				Expect(err).ToNot(HaveOccurred())
 			})
+			It("[test_id:XXXX]CDI CR deletion should delete DataImportCron CRD and all DataImportCrons with finalizers", func() {
+				By("Create new DataImportCron")
+				url := fmt.Sprintf(utils.TrustedRegistryURL, f.DockerPrefix)
+				cron := NewDataImportCron("cron-test", "5Gi", scheduleEveryMinute, "ds", cdiv1.DataVolumeSourceRegistry{URL: &url, PullMethod: &registryPullNode}, cdiv1.DataImportCronRetainAll)
+				cron, err := f.CdiClient.CdiV1beta1().DataImportCrons(f.Namespace.Name).Create(context.TODO(), cron, metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
can we wait for DV to be created/succeed?  Or at least validate that the finalizer exists?

Reply:
fixed